langlib/lang.float/src/main/ballerina/float.bal

```diff
@@ -78,6 +103,16 @@ public isolated function sum(float... xs) returns float = @java:Method {
 # Result is -∞ if no args
 # NaN if any arg is NaN
```

Comment:
Do we need to add examples for these scenarios?
Imo, no need to add.

Reply:
We can demonstrate them with simple (one-line) examples, right? So +1 to add.